zzupy/model/eas.py

```
@@ -0,0 +1,778 @@
+        if not isinstance(data, dict):
+            return data
+
+        start_date_str = data.get("start_date")
+        end_date_str = data.get("end_date")
+        if not start_date_str and not end_date_str:
+            return data
+
+        try:
+            data["start_date"] = Date.parse_iso(start_date_str)
+            data["end_date"] = Date.parse_iso(end_date_str)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)